daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -498,37 +449,44 @@ func (l *Libtelio) Refresh(c mesh.MachineMap) error {
 		return nil
 	}
 
-	meshnetMap := string(c.Raw)
+	config, err := teliogo.DeserializeMeshnetConfig(string(c.Raw))
+	if err != nil {
+		return fmt.Errorf("failed to deserialize meshnet config: %w", err)
+	}
 
-	result := teliogo.TELIORESOK
+	err = nil
```

Comment:
[minor] prob this is not needed since it cannot get here unless is already `nil`?